Makefile

```diff
@@ -50,16 +50,24 @@ clean:
 	rm -rf dist
 
 .PHONY: dist
-dist: clean browserpass-linux64 browserpass-darwinx64 browserpass-openbsd64 browserpass-freebsd64 browserpass-windows64
+dist: clean browserpass-linux64 browserpass-darwin64 browserpass-openbsd64 browserpass-freebsd64 browserpass-windows64
 	mkdir -p dist
 
 	git archive -o dist/$(VERSION).tar.gz --format tar.gz --prefix=browserpass-native-$(VERSION)/ $(VERSION)
 
-	zip -FSr dist/browserpass-linux64-$(VERSION).zip   browserpass-linux64       browser-files/* Makefile README.md LICENSE
-	zip -FSr dist/browserpass-darwinx64-$(VERSION).zip browserpass-darwinx64     browser-files/* Makefile README.md LICENSE
-	zip -FSr dist/browserpass-openbsd64-$(VERSION).zip browserpass-openbsd64     browser-files/* Makefile README.md LICENSE
-	zip -FSr dist/browserpass-freebsd64-$(VERSION).zip browserpass-freebsd64     browser-files/* Makefile README.md LICENSE
-	zip -FSr dist/browserpass-windows64-$(VERSION).zip browserpass-windows64.exe browser-files/* Makefile README.md LICENSE
+	$(eval TMP := $(shell mktemp -d))
+
+	for os in linux64 darwin64 openbsd64 freebsd64 windows64; do \
+	    mkdir $(TMP)/browserpass-"$$os"-$(VERSION); \
+	    cp -a browserpass-"$$os"* browser-files Makefile README.md LICENSE $(TMP)/browserpass-"$$os"-$(VERSION); \
+	    if [ "$$os" = "windows64" ]; then \
+	        (cd $(TMP) && zip -FSr ${CURDIR}/dist/browserpass-"$$os"-$(VERSION).zip browserpass-"$$os"-$(VERSION)); \
+	    else \
+	        (cd $(TMP) && tar -cvf ${CURDIR}/dist/browserpass-"$$os"-$(VERSION).tar.gz --use-compress-program=gzip browserpass-"$$os"-$(VERSION)); \
```

Comment:
I'm curious why `--use-compress-program=gzip` rather than simply using `-z`. It doesn't matter either way, and the result is identical, but the choice of syntax is unusual.

Reply:
I don't usually use `tar` directly so I simply didn't know about `-z` flag, I use a helper script from prezto, so I took the command from their sources - and their command looks like `--use-compress-program=pigz` because they use parallel compression 🙂 

But for this Makefile will simplify, since archives are small and I wasn't planning to use parallel compression here anyway.